stable/parse/templates/server-deployment.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
```

Comment:
Add `component: "{{ .Chart.Name }}"` label.

For consistency, how about using `name: {{ template "fullname" . }}-server` and `component: "{{ .Chart.Name }}-server"`. Actually, I would not even add the chart name to the components at all.